nmtrust

```diff
@@ -70,26 +81,31 @@ while getopts ":f:qh" opt; do
     esac
 done
 
-# Check if the trusted network file exists.
+# Check if the excluded and trusted networks files exist.
 file_check
 
 # Get all active connections.
-connections=($(nmcli --terse -f uuid conn show --active))
+connections=($(nmcli --terse -f name,uuid conn show --active))
+
+# Get number of active connections.
+num_connections=$(comm -13 <(sort "$EXCLUDEFILE") <(nmcli --terse -f name conn show --active | sort) | wc -l)
```

Comment:
This also should not depend on the exclusion file being specified. Perhaps up above, where the existence of the file is checked if the var is set, we read the sorted contents into a variable which is otherwise set to an empty string. Down here the variable can be used without caring about its contents.

Reply:
(Unless `EXCLUDEFILE` defaults to `/dev/null`)

Reply:
See check_file